packages/node/src/indexer/api.service.ts

```diff
@@ -93,13 +92,7 @@ export class ApiService implements OnApplicationShutdown {
     if (this.patchedApi) {
       return this.patchedApi;
     }
-    // TODO: remove once https://github.com/polkadot-js/api/pull/3949 is merged
-    const {
-      network: { endpoint },
-    } = this.project;
-    const patchedApi = endpoint.startsWith('ws')
-      ? this.getApi().clone()
-      : new ApiPromise(this.apiOption);
+    const patchedApi = this.getApi().clone();
```

Comment:
Need to keep it as it is for now. The release contains this PR is `@polkadot/api@6.0.1`, we are still using v5.9.1